mkosi

```diff
@@ -394,8 +394,23 @@ def enable_networkd(workspace):
 def install_fedora(args, workspace, run_build_script):
     print_step("Installing Fedora...")
 
+    with open(os.path.join(workspace, "dnf.conf"), "w") as f:
+        f.write("[main]\n")
+        f.write("gpgcheck=1\n")
+        f.write("\n")
+        f.write("[fedora]\n")
+        f.write("name=Fedora %s - base\n" % args.release)
+        f.write("baseurl=https://mirrors.kernel.org/fedora/releases/%s/Everything/x86_64/os/\n" % args.release)
+        f.write("gpgkey=https://getfedora.org/static/81B46521.txt\n")
+        f.write("\n")
+        f.write("[updates]\n")
+        f.write("name=Fedora %s - updates\n" % args.release)
+        f.write("baseurl=https://mirrors.kernel.org/fedora/updates/%s/x86_64/\n" % args.release)
+        f.write("gpgkey=https://getfedora.org/static/81B46521.txt\n")
```

Comment:
btw, i think it would be nicer if we'd start using multine strings for this.

I can't say I like the hardcoding of the x86_64 thing, we should probably make it match the host arch... but then again I can't say I care too much for now, after all pretty much everything real is x86_64 these days...